pkg/publicapi/endpoint/orchestrator/job.go

```diff
@@ -114,13 +115,26 @@ func (e *Endpoint) getJob(c echo.Context) error {
 				return err
 			}
 			response.Executions = &apimodels.ListJobExecutionsResponse{
-				Executions: make([]*models.Execution, len(executions)),
+				Items: make([]*models.Execution, len(executions)),
 			}
 			for i := range executions {
-				response.Executions.Executions[i] = &executions[i]
+				response.Executions.Items[i] = &executions[i]
 			}
 		}
 	}
+
+	clientVersion := apimodels.GetClientVersion(c.Request())
+	if !(clientVersion.GreaterThan(version.V1_3_2) || clientVersion.Equal(version.Development) || clientVersion.Prerelease() != "") {
+		if response.History != nil && len(response.History.History) > 0 { //nolint: staticcheck
```

Comment:
Why do we need this check?

Reply:
we don't I've adjusted in latest commit.